unit-tests/live/d500/test-dds-embedded-filters.py

```diff
@@ -7,9 +7,39 @@
 import pyrealsense2 as rs
 from rspy import test, log
 
+def check_option_in_list(option_id, options_list):
+    for option in options_list:
+        if option_id == option:
+            return True
+    return False
+
+def set_get_filter_option_value(embedded_filter, option, value_to_assign):
+    initial_value = embedded_filter.get_option(option)
+    option_step =  embedded_filter.get_option_range(option).step
+    embedded_filter.set_option(option, value_to_assign)
+    test.check_approx_abs(embedded_filter.get_option(option), value_to_assign, option_step)
+    embedded_filter.set_option(option, initial_value)
+    test.check_approx_abs(embedded_filter.get_option(option), value_to_assign, option_step)
+
+def parse_fw_version(version_str):
```

Comment:
See this comment:
https://github.com/IntelRealSense/librealsense/pull/14377/files#r2469590883